### PR TITLE
[12.0][ADD] mrp_production_quant_manual_assign

### PR DESCRIPTION
**Pull request by yostashiro**
This module just adjusts the manufacturing order form to add a link in the component lines to the wizard from https://github.com/OCA/stock-logistics-warehouse/tree/12.0/stock_quant_manual_assign.

### CODE REVIEW

**Review by TimLai125 — APPROVED** [review]
LGTM!

**Review by marcelsavegnago — APPROVED** [review]
LGTM

File: mrp_production_quant_manual_assign/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Quartile Limited
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Production - Manual Quant Assignment",
+    "version": "12.0.1.0.0",
+    "category": "Manufactureing",
```
Comment (marcelsavegnago) [inline mrp_production_quant_manual_assign/__manifest__.py]:
```suggestion
    "category": "Manufacturing",
```

Comment (yostashiro) [reply mrp_production_quant_manual_assign/__manifest__.py]: @marcelofrare Thank you! Fixed.

**Pull request merged**